.github/workflows/doc-build.yml

```diff
@@ -124,7 +124,7 @@ jobs:
           set -ex
           export SAGE_USE_CDNS=yes
           mv /sage/local/share/doc/sage/html/en/.git /sage/.git-doc
-          make doc-clean doc-uninstall sagelib-clean && git clean -fx src/sage
+          make doc-clean doc-uninstall
           mkdir -p /sage/local/share/doc/sage/html/en/ && mv /sage/.git-doc /sage/local/share/doc/sage/html/en/.git
           ./config.status && make doc-html
```

Comment:
Not relevant here, but I wonder what `/sage/local/share/doc/sage/html/en/.git` is, and why it is important to preserve this across `doc-clean doc-uninstall`.

Reply:
This is how we keep track of the changes in the built documentation, compared to what is contained in the Docker image.

Reply:
Ah, I see. You use this information to create "changed" files in the doc.

Reply:
By the way, the "changes" list seems not working properly...

Reply:
> the "changes" list seems not working properly...

This is from a recent change to CDN use for mathjax -- see the diff: https://deploy-preview-36473--sagemath-tobias.netlify.app/html.diff

Reply:
I've opened https://github.com/sagemath/sage/pull/36475 for fixing this